celery/platforms.py

```diff
@@ -255,11 +255,16 @@ def create_pidlock(pidfile):
     return pidlock
 
 
-def _create_pidlock(pidfile):
+def _free_pidfile_or_die(pidfile):
     pidlock = Pidfile(pidfile)
-    if pidlock.is_locked() and not pidlock.remove_if_stale():
-        print(PIDLOCKED.format(pidfile, pidlock.read_pid()), file=sys.stderr)
+    if pidlock.is_locked():
+        print(PIDLOCKED.format(pidlock, pidlock.read_pid()), file=sys.stderr)
```

Comment:
Does the `pidlock` object representation contain more information?

I wouldn't advise changing something like this, as users may have already built some kind of alerting mechanism around this message pattern.